README.md

```diff
@@ -4,6 +4,21 @@
 
 **QA: Shreyas Sabnis**
 
```

Comment:
Instructions to setup environment variables could be incuded

Reply:
I forgot to update my requirements.txt. It is now replaced with a file environment.yml

Reply:
The environment.yml works perfectly! However the create_db_rds script has this:
user = os.environ.get("MYSQL_USER")
password = os.environ.get("MYSQL_PASSWORD")
Since these are environment variables, it might be helpful to include instructions on how to add them to the users' bash profile

Reply:
You have great instructions for that from your project and I emulated how you did it. That's fantastic. Thanks!